eos-upgrade-eos2-to-eos3

```diff
@@ -0,0 +1,222 @@
+#!/bin/bash -e
+
+# Upgrade from eos2 to eos3
+
+# Removes all installed legacy app bundles,
+# points to the latest ostree branches,
+# and starts the ostree upgrade process.
+# Requires the user to re-install any desired
+# apps as flatpaks from the app center
+# after rebooting into eos3.
+
+ARGS=$(getopt -o fsh -l "force,skip,help" -n "$0" -- "$@")
+eval set -- "$ARGS"
+
+usage() {
+    cat <<EOF
+Remove all legacy apps and upgrade from eos2 to eos3
+Usage:
+   $0
+Options:
+   -f,--force	don't ask for confirmation
+   -s,--skip    skip the final step to actually perform the ostree upgrade
+   -h,--help	show this message
+EOF
+}
+
+FORCE=false
+SKIP=false
+while true; do
+    case "$1" in
+        -f|--force)
+            FORCE=true
+            shift
+            ;;
+        -s|--skip)
+            SKIP=true
+            exit 0
```

Comment:
Oh, that's silly.  I must have copied the "-h" below as a starting point.  Yes, should be a "shift" here instead.